sdk_container/src/third_party/coreos-overlay/coreos-base/misc-files/misc-files-0.ebuild

```
@@ -92,4 +109,23 @@ src_install() {
         dosym "${target}" "${link}"
         fowners --no-dereference 500:500 "${link}"
     done
+
+    if use openssh; then
+        # Install our configuration snippets.
+        insinto /etc/ssh/ssh_config.d
+        doins "${FILESDIR}/50-flatcar-ssh.conf"
+        insinto /etc/ssh/sshd_config.d
+        doins "${FILESDIR}/50-flatcar-sshd.conf"
+
+        # Install our socket drop-in file that disables the rate
+        # limitting on the sshd socket.
```

I'll fix this on the next push:
```suggestion
        # limiting on the sshd socket.
```